<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>add title</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(241, 231, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="grid_layout_left">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="vertical_layout_left">
        <item>
         <layout class="QHBoxLayout" name="horizontal_layout_title">
          <item>
           <widget class="QPushButton" name="back_button">
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color: rgb(230, 208, 255);
	border-radius: 15px;
	padding: 5px;
    width: 20px;
    height: 20px;
}
QPushButton:hover{
	background-color: rgb(157, 0, 255);
}</string>
            </property>
            <property name="text">
             <string>&lt;-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="title_text">
            <property name="text">
             <string>title:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="title_input">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontal_layout_comment">
          <item>
           <widget class="QLabel" name="input_comment_text">
            <property name="text">
             <string>введите отзыв:     </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="comment_input">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="vertical_layout_right">
        <item>
         <layout class="QGridLayout" name="grid_layout_types">
          <item row="0" column="1">
           <widget class="QRadioButton" name="r2">
            <property name="styleSheet">
             <string notr="true">QRadioButton{
	background-color: rgb(241, 231, 255);
	border-radius: 10px;
	padding: 5px;
    width: 70px;
    height: 15px;
}
QRadioButton:hover{
	background-color: rgb(255, 148, 218);
}</string>
            </property>
            <property name="text">
             <string>book</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="r1">
            <property name="styleSheet">
             <string notr="true">QRadioButton{
	background-color: rgb(241, 231, 255);
	border-radius: 10px;
	padding: 5px;
    width: 70px;
    height: 15px;
}
QRadioButton:hover{
	background-color: rgb(255, 148, 218);
}</string>
            </property>
            <property name="text">
             <string>film</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QRadioButton" name="r8">
            <property name="styleSheet">
             <string notr="true">QRadioButton{
	background-color: rgb(241, 231, 255);
	border-radius: 10px;
	padding: 5px;
    width: 70px;
    height: 15px;
}
QRadioButton:hover{
	background-color: rgb(255, 148, 218);
}</string>
            </property>
            <property name="text">
             <string>comic strip</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QRadioButton" name="r9">
            <property name="styleSheet">
             <string notr="true">QRadioButton{
	background-color: rgb(241, 231, 255);
	border-radius: 10px;
	padding: 5px;
    width: 70px;
    height: 15px;
}
QRadioButton:hover{
	background-color: rgb(255, 148, 218);
}</string>
            </property>
            <property name="text">
             <string>cartoon</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="r4">
            <property name="styleSheet">
             <string notr="true">QRadioButton{
	background-color: rgb(241, 231, 255);
	border-radius: 10px;
	padding: 5px;
    width: 70px;
    height: 15px;
}
QRadioButton:hover{
	background-color: rgb(255, 148, 218);
}</string>
            </property>
            <property name="text">
             <string>anime</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QRadioButton" name="r6">
            <property name="styleSheet">
             <string notr="true">QRadioButton{
	background-color: rgb(241, 231, 255);
	border-radius: 10px;
	padding: 5px;
    width: 70px;
    height: 15px;
}
QRadioButton:hover{
	background-color: rgb(255, 148, 218);
}</string>
            </property>
            <property name="text">
             <string>manhua</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="r7">
            <property name="styleSheet">
             <string notr="true">QRadioButton{
	background-color: rgb(241, 231, 255);
	border-radius: 10px;
	padding: 5px;
    width: 70px;
    height: 15px;
}
QRadioButton:hover{
	background-color: rgb(255, 148, 218);
}</string>
            </property>
            <property name="text">
             <string>series</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QRadioButton" name="r3">
            <property name="styleSheet">
             <string notr="true">QRadioButton{
	background-color: rgb(241, 231, 255);
	border-radius: 10px;
	padding: 5px;
    width: 70px;
    height: 15px;
}
QRadioButton:hover{
	background-color: rgb(255, 148, 218);
}</string>
            </property>
            <property name="text">
             <string>manhwa</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="r5">
            <property name="styleSheet">
             <string notr="true">QRadioButton{
	background-color: rgb(241, 231, 255);
	border-radius: 10px;
	padding: 5px;
    width: 70px;
    height: 15px;
}
QRadioButton:hover{
	background-color: rgb(255, 148, 218);
}</string>
            </property>
            <property name="text">
             <string>mannga</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontal_layout_rating_status">
          <item>
           <widget class="QLabel" name="rating_text">
            <property name="text">
             <string>rating</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="rating_spin_box">
            <property name="styleSheet">
             <string notr="true">QSpinBox{
	background-color: rgb(230, 208, 255);
	border-radius: 10px;
	padding: 5px;
    width: 20px;
    height: 15px;
}
QSpinBox:hover{
	background-color: rgb(157, 0, 255);
}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="rating_from">
            <property name="text">
             <string>/10          </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="status_text">
            <property name="text">
             <string>status</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="status_combo_box">
            <property name="styleSheet">
             <string notr="true">QComboBox{
	background-color: rgb(230, 208, 255);
	border-radius: 10px;
	padding: 5px;
    width: 80px;
    height: 15px;
}
QComboBox:hover{
	background-color: rgb(157, 0, 255);
}</string>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
            <item>
             <property name="text">
              <string>planned</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>in progress</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>finished</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="vertical_layout_add">
          <item>
           <layout class="QHBoxLayout" name="horizontal_layout_add">
            <item>
             <spacer name="spacer_add_left">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="button_add">
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color: rgb(230, 208, 255);
	border-radius: 10px;
	padding: 5px;
    width: 60px;
    height: 15px;
}
QPushButton:hover{
	background-color: rgb(157, 0, 255);
}</string>
              </property>
              <property name="text">
               <string>add</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer_add_right">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
